fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,164 @@
+import time
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from typing import Any
+
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+metrics_list = ['accuracy']
+
+
+def create_tags(**tags_in):
+    tags = []
+
+    tags.append("project:" + tags_in['project'])
+    tags.append("model:" + tags_in['model'])
+    if len(tags_in) > 2:
+        tags.append("feature:" + tags_in['feature'])
+
+    return tags
+
+
+class FiddlerCheck(AgentCheck):
+
+    # This will be the prefix of every metric and service check the integration sends
+    __NAMESPACE__ = 'fiddler'
+
+    def __init__(self, name, init_config, instances):
+        super(FiddlerCheck, self).__init__(name, init_config, instances)
+
+        # Connection parameters for Fiddler. Will be parameterizing this later.
+        self.URL = 'https://demo.fiddler.ai'
+        self.ORG_ID = 'demo'
+        self.AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
+
+        self.log.info("Inside init method.")
+        #        self.client = fdl.FiddlerApi(url=self.URL, org_id=self.ORG_ID, auth_token=self.AUTH_TOKEN)
+
+        self.base_url = self.instance.get('url')
+        self.api_key = self.instance.get('fiddler_api_key')
+        self.org = self.instance.get('organization')
+
+        #            self.log.info("Connecting to : ", self.base_url)
+        #            self.log.info("with org id : ", self.org)
+        #            self.log.info("and auth key : ", self.api_key)
+
+        print("Connecting to : ", self.base_url)
+        print("with org id : ", self.org)
+        print("and auth key : ", self.api_key)
+
+        self.client = fdl.FiddlerApi(url=self.base_url, org_id=self.org, auth_token=self.api_key)
+
+    # Use self.instance to read the check configuration
+
+    def check(self, _):
+        # Iterate through the projects and the models and push data into Fiddler
+        project_path = ['list_projects', self.ORG_ID]
+        result_all = self.client.v1._call(project_path)
+        #        self.log.info("Projects: ", result_all["projects"])
+
+        start_time = (time.time() * 1000) - (bin_size * 1000)
+        end_time = time.time() * 1000
+        self.log.info("Start time is : ", start_time)
+        self.log.info("End time is : ", end_time)
```

Comment:
Given the nature of these log messages, I think it would be better to turn `self.log.info`'s to `self.log.debug`'s (in the whole code).

```suggestion
        self.log.debug("Start time is: %s", start_time)
        self.log.debug("End time is: %s", end_time)
```

Reply:
Done.